From 86e27cdbf119440101a850e1b44baebfefe863ca Mon Sep 17 00:00:00 2001
From: Martin Sandsmark <martin.sandsmark@kde.org>
Date: Thu, 5 Jul 2012 09:40:07 +0200
Subject: [PATCH 1/1] hctl: Remove a useless assert

The case where the element is unavailable (for example gone away since
the event was created) is handled beneath.

See also bug 5471.
	https://bugtrack.alsa-project.org/alsa-bug/view.php?id=5471

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 src/control/hcontrol.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/src/control/hcontrol.c b/src/control/hcontrol.c
index 8ffc434..ee1d907 100644
--- a/src/control/hcontrol.c
+++ b/src/control/hcontrol.c
@@ -761,7 +761,6 @@ static int snd_hctl_handle_event(snd_hctl_t *hctl, snd_ctl_event_t *event)
 	if (event->data.elem.mask & (SNDRV_CTL_EVENT_MASK_VALUE |
 				     SNDRV_CTL_EVENT_MASK_INFO)) {
 		elem = snd_hctl_find_elem(hctl, &event->data.elem.id);
-		assert(elem);
 		if (!elem)
 			return -ENOENT;
 		res = snd_hctl_elem_throw_event(elem, event->data.elem.mask &
-- 
1.7.10.4

